### PR TITLE
Add reset REPL command

### LINKED ISSUE #142: Introduce command in repl that clears defined or loaded bindings
Currently I have to restart the REPL to achieve the same effect. Restarting the REPL-session could be prevented by such command.

### PR DESCRIPTION
**Pull request by rexim**
Close #142

### CODE REVIEW

**Comment by keddelzz** [comment]
LGTM :)

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 # Morganey [![Status Ventis](https://img.shields.io/badge/status-ventis-yellow.svg)](https://github.com/ForNeVeR/andivionian-status-classifier)
 
-It's the [Lambda Calculus][wiki-lambda-calculus] Interpreter and it
-interprets lambda terms! \o/
+[Lambda Calculus][wiki-lambda-calculus] based programming language
+that recognises Church encoded structures.
```
Comment (ForNeVeR) [inline README.md]: You mix British English (e.g. this `recognise` instance) and American English (e.g. `favorite` below). My vote for American dialect, so I'd replace this `recognise` with `recognize`.

**Comment by ForNeVeR** [comment]
LGTM. :+1:

File: README.md (hunk 1 above)
Comment (rexim) [reply README.md]: I don't usually care about English dialects, but sure let's be consistent here.

**Comment by rexim** [comment]
Thank you guys for the review!

**Pull request merged**